src/main/kotlin/com/ampnet/blockchainapiservice/controller/ContractDeploymentRequestController.kt

```diff
@@ -76,6 +76,19 @@ class ContractDeploymentRequestController(
         )
     }
 
+    @GetMapping("/v1/deploy/by-project/{projectId}/by-alias/{alias}")
+    fun getContractDeploymentRequestByProjectIdAndAlias(
```

Comment:
Move this function above `getContractDeploymentRequestsByProjectIdAndFilters` - because `@GetMapping`s have priority by function order, so we want to make this one have higher priority than `"/v1/deploy/by-project/{projectId}"` because `{projectId}` can capture `/by-alias/{alias}` part of the path too.